dom.bs

```diff
@@ -2389,16 +2389,13 @@ of a <var>node</var> into a <var>parent</var> before a <var>child</var>, run the
 
 <p><a lt="Other applicable specifications">Specifications</a> may define
 <dfn export id=concept-node-insert-ext>insertion steps</dfn> for all or some <a for=/>nodes</a>. The
-algorithm is passed <var ignore>insertedNode</var>, as indicated in the <a for=/>insert</a>
-algorithm below.
-<!-- See https://github.com/whatwg/dom/issues/34#issuecomment-125571750 for why we might need to
-     adjust this further based on the requirements of the script element. There might be other ways
-     to define that though as Olli suggests, so leaving that out for now. -->
+algorithm is passed <var ignore>insertedNode</var> and <var ignore>deferredStepsQueue</var>, as
+indicated in the <a for=/>insert</a> algorithm below.
 
 <p><a lt="other applicable specifications">Specifications</a> may define
 <dfn export id=concept-node-children-changed-ext>children changed steps</dfn> for all or some
-<a for=/>nodes</a>. The algorithm is passed no argument and is called from <a for=/>insert</a>,
-<a for=/>remove</a>, and <a for=/>replace data</a>.
+<a for=/>nodes</a>. The algorithm is passed <var ignore>deferredStepsQueue</var> and is called from
```

Comment:
Addressing https://github.com/whatwg/html/pull/4354#issuecomment-476038918 by @tkent-google in a way that aligns with Chrome would also require passing "insert"/"change"/"remove" as an argument here and acting on it accordingly. (Script would ignore anything but "insert".)

Probably would have to double check whether setting `textContent` counts as "change" or "insert".